src/infohud.asm

```diff
@@ -586,6 +581,136 @@ status_cooldowncounter:
     RTS
 }
 
+status_shinefinetune:
```

Comment:
This method is organized to reduce JMP statements.  I only ended up need one.  It's nice that the compiler tells us when the relative branch jumps are too long.